region/wreckedship/main/Sponge Bath.json

```diff
@@ -254,7 +254,7 @@
       "name": "Midair SpringBall",
       "requires": [
         "canSuitlessMaridia",
-        "canSpringBallJumpMidAir"
+        "canTrickySpringBallJump"
```

Comment:
not that it needs to be done now, but a lot of these need a way to deal with the Bull